deployments/with-creds/ci/values.yaml

```diff
@@ -32,6 +32,27 @@ concourse:
                 app: ci-web
                 release: ci
     sidecarContainers:
+      - name: telegraf
+        image: telegraf
+        volumeMounts:
+        - name: dsdsocket
+          mountPath: /var/run/datadog
+        command:
+        - /bin/bash
+        - -c
+        - |
+          echo '
+          [[inputs.prometheus]]
```

Comment:
Would it make sense to pull this into a config map ?

Reply:
frankly i prefer seeing all the config in one place -- the only reason i didn't do something like this for the opencensus agent is because it's such a lightweight image - it doesn't actually have a shell so i can't do the same `exec` trick. The problem with the configmap strategy is the usual k8s/helm struggle -- there's nothing in k8s saying that the container needs to restart (and volume get recreated) when the configmap changes.

Reply:
Agreed that its easier to look at it inline, but I feel having a consistent pattern has more utility. Either way, since this would only be there for a the transition period, its not a big deal.